### DISCUSSION TITLE
How to use contents.getPrintersAsync() from Electron?

### DISCUSSION THREAD

**Discussion by elfeffe** [NativePHP/laravel#148] [Q&A] [answered]
I need to use this function https://www.electronjs.org/es/docs/latest/api/web-contents#contentsgetprintersasync
I have been trying many different ways, but I always get an error in the console.

**Comment by simonhamp** [comment, marked as answer]
Good news! [We already do](https://github.com/NativePHP/electron-plugin/blob/main/src/server/api/system.ts#L22-L28) and you can use this from the Laravel/PHP side [via the `System` facade](https://github.com/NativePHP/laravel/blob/main/src/System.php#L29-L43)
